archive-tar.c

```diff
@@ -38,9 +38,9 @@ static int write_tar_filter_archive(const struct archiver *ar,
 #endif
```

Comment:
The commit message probably wants to be improved a little, e.g. by starting the oneline with "archive:" and talking more about the why? instead of the how?. Like, for example, "archive: avoid spawning `gzip`", and then continuing by explaining that MinGit for Windows ships without `gzip`, which is perfectly legitimate, and we should not even require that executable to produce `.tar.gz` files because we *already* link to libz so we can do all this ourselves.